apps/aecore/src/aec_peer_connection.erl

```diff
@@ -414,21 +432,28 @@ handle_msg(S, _MsgType, true, {RequestFld, From, _TRef}, {error, Reason}) ->
 handle_msg(S, MsgType, false, Request, {ok, Vsn, Msg}) ->
     case MsgType of
         ping                 -> handle_ping(S, Request, Msg);
-        get_mempool          -> handle_get_mempool(S, Msg);
         get_header_by_hash   -> handle_get_header_by_hash(S, Msg);
         get_header_by_height -> handle_get_header_by_height(S, Vsn, Msg);
         get_n_successors     -> handle_get_n_successors(S, Vsn, Msg);
         get_block            -> handle_get_block(S, Msg);
         block                -> handle_new_block(S, Msg);
-        tx                   -> handle_new_tx(S, Msg)
+        txs                  -> handle_new_txs(S, Msg);
+        txps_init            -> handle_tx_pool_sync_init(S, Msg);
+        txps_unfold          -> handle_tx_pool_sync_unfold(S, Msg);
+        txps_get             -> handle_tx_pool_sync_get(S, Msg);
+        txps_finish          -> handle_tx_pool_sync_finish(S, Msg)
     end;
 handle_msg(S, MsgType, true, Request, {ok, _Vsn, Msg}) ->
     case MsgType of
-        ping    -> handle_ping_rsp(S, Request, Msg);
-        mempool -> handle_get_mempool_rsp(S, Request, Msg);
-        header  -> handle_header_rsp(S, Request, Msg);
+        ping          -> handle_ping_rsp(S, Request, Msg);
+        header        -> handle_header_rsp(S, Request, Msg);
         header_hashes -> handle_header_hashes_rsp(S, Request, Msg);
-        block   -> handle_get_block_rsp(S, Request, Msg)
+        block         -> handle_get_block_rsp(S, Request, Msg);
+        txps_init     -> handle_tx_pool_sync_rsp(S, init, Request, Msg);
+        txps_unfold   -> handle_tx_pool_sync_rsp(S, unfold, Request, Msg);
+        txs           -> handle_tx_pool_sync_rsp(S, get, Request, Msg);
```

Comment:
I would expect to see txps_get here instead?

Reply:
Why? The response to a `txps_get` is a `txs` The others happen to have the same response as the request...